### PR TITLE
RD-2363 Remove unused pages

### PR DESCRIPTION
**Pull request by leon-good-life**
## Description
Jira ticket: https://cloudifysource.atlassian.net/browse/RD-2363

I checked the list of pages used in community and premium:
- adminDash
- blueprints
- deployments
- executions
- resources

Main sys admin has also:
- systemSetup

I made a list of drill-down pages:
- execution
- blueprint
- deployment
- logs
- blueprintMarketplace
- deploy
- drilldownDeployments

I made a list of pages inside page-groups:
- services
- environments
- secrets
- plugins
- sites
- agents
- filters
- tokens
- users
- groups
- tenants
- health
- logs
- snapshots

If I understood correctly the code, Other pages that not described above should be removed.

**Removed:**
1. `ha.json` there is no reference to it in the code.

**Maybe I should remove those files:**
1. I don't see reference in the code for `systemResources.json`.
2. There is a reference to `tmm.json` only in a test file.

In `templates/pages` directory, all pages except 3 are used (and I listed them). One page I removed. The other two I'm not sure if I need to remove.

## Screenshots / Videos
N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
I started `templates_spec` on Jenkins: https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/2975/pipeline

 

## Documentation
N/A

### CODE REVIEW

**Comment by leon-good-life** [comment]
In templates/pages directory, all pages except 3 are used (and I listed them in the PR description). One page I removed. The other two I'm not sure if I need to remove.

I Removed:
- ha.json there is no reference to it in the code.

Maybe I should remove those files:
- I don't see reference in the code for systemResources.json.
- There is a reference to tmm.json only in a test file.

**Comment by kubama** [comment]
> I don't see reference in the code for systemResources.json.

So why it's not removed?

> There is a reference to tmm.json only in a test file.

I would remove the reference and drop the file

**Comment by leon-good-life** [comment]
I started running system tests on Jenkins: https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/2954/pipeline

**Comment by leon-good-life** [comment]
I started system tests after changes: https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/2962/pipeline

File: test/cypress/integration/templateManagement/templates_spec.ts [hunk 1]
```diff
@@ -141,9 +141,6 @@ describe('Templates segment', () => {
             cy.get('[option-value="T2"]').click();
             clickOnHeader();
 
-            cy.log('Add pages');
-            cy.contains('Available pages').click().parent().contains('tmm').find('.add').click();
-
```
Comment (kubama) [inline test/cypress/integration/templateManagement/templates_spec.ts:146]: Can we use a different page instead of removing the test step?

**Comment by leon-good-life** [comment]
I updated ``templates_spec`, I verified it passed on my machine. I started `templates_spec` on Jenkins: https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/2975/pipeline

File: test/cypress/integration/templateManagement/templates_spec.ts (hunk 1 above)
Comment (leon-good-life) [reply test/cypress/integration/templateManagement/templates_spec.ts]: I changed to `users` page.

**Review by kubama — APPROVED** [review]

**Pull request merged**